css-simple-length.html

```diff
@@ -0,0 +1,59 @@
+<script>
+
+function calcSingleStats(times) {
```

Comment:
This is from the "main.html". Maybe we can make another JS file to have this function (and maybe other functions) later